src/plugins/visualize/public/application/components/visualize_top_nav.tsx

```diff
@@ -216,14 +259,15 @@ const TopNav = ({
       savedQueryId={currentAppState.savedQuery}
       onSavedQueryIdChange={stateContainer.transitions.updateSavedQuery}
       indexPatterns={indexPatterns}
-      screenTitle={vis.title}
+      screenTitle={vis.title.length ?? '' ? vis.title : 'New Visualization'}
```

Comment:
I believe this should be
```suggestion
      screenTitle={vis.title.length ?? '' ? vis.title : 'New visualization'}
```

Reply:
Thanks. Updated it